src/client/components/UsersList.tsx

```diff
@@ -7,18 +7,40 @@ import ListItem from '@mui/material/ListItem';
 import React from 'react';
 import { remult } from '../utils/api-facade';
 import { User } from '../../shared/User';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Button, IconButton, Stack } from '@mui/material';
+import { UserEditor } from './UserEditor';
 
 const userRepo = remult.repo(User);
 
 export const UsersList: React.FC = () => {
   const [users, setUsers] = React.useState<User[]>(null);
   const [isLoading, setIsLoading] = React.useState(true);
+  const [editUser, setEditUser] = React.useState<User>(null);
+  const [newUser, setNewUser] = React.useState<User>(null);
+
+  const editUserSaved = (editUser: User) =>
+    setUsers(users.map(user => user.userId === editUser.userId ? editUser : user));
+
+  const newUserSaved = (newUser: User) =>
+    setUsers([...users, newUser]);
+  const deleteUser = async (user: User) => {
+    await userRepo.delete(user);
+    setUsers(users.filter(u => u.userId != user.userId));
```

Comment:
_Look of disapproval_
```suggestion
    setUsers(users.filter(u => u.userId !== user.userId));
```